packages/logic-utils/src/utils/flatten-members.ts

```diff
@@ -10,7 +10,13 @@ function flattenMembers<T>(
   if (ui5SuperClass !== undefined) {
     // UI5 SDK refers to inherited members (properties, events, aggregations. ...) as "borrowed" ...
     const borrowedMembers = flattenMembers(membersGetter, ui5SuperClass);
-    return directMembers.concat(borrowedMembers);
+    const borrowedMembersWithoutOverrides = filter(
```

Comment:
Should we removed the borrowed that are also "overrides"
Or should we remove the original definitions that have been overridden?

Which is more correct? and does it even matter?

Reply:
We should remove original definitions that have been overridden. There is also documentation information in the completion items, which may differ.

Reply:
Are not we doing the opposite here by removing the "borrowed" && "overriden"
instead of "original" && "overriden"?

Or am I missing something here?

Reply:
With "original" definition I meant the ones that are coming from the parent class, maybe my explanation was misleading. If we have the following example:
```Typescript
interface A {
   one: () => void;
   two: string;
}
interface B extends A {
   one: () => void;
   three: string;
}
```

When considering `B` the function `one` from `A` should be removed and the `one` from `B` should be used. The code here should be doing that as far as I can see.